### DISCUSSION TITLE
Is select(dbid) thread safe on redis cluster mode?

### DISCUSSION THREAD

**Discussion by shrimptttttttttt** [redis/lettuce#2454] [Q&A] [answered]
If there are two threads calling select with different dbid on redis cluster mode, which means there are multiple instances, what will happen? Is it thread-safe?

**Comment by mp911de** [comment, marked as answer]
No because Cluster does not have databases.

**Reply by shrimptttttttttt** [reply]
OK, got it, thanks.
